fitnessllm_dataplatform/stream/strava/etl_utils.py

```
@@ -0,0 +1,243 @@
+        for element in input_dict["data"]:
+            element["athlete_id"] = input_dict["athlete_id"]
+            element["activity_id"] = input_dict["activity_id"]
+            for k, v in element.items():
+                if type(v) in [list]:
+                    element[k] = str(v)
+        return input_dict["data"]
+    return {}
+
+
+def load_json_into_dataframe(
+    athlete_id: str, file: GSPath, data_source, data_stream: FitnessLLMDataStream
```

Ruff (0.8.2)

Missing type annotation for function argument `data_source`

(ANN001)